src/redux/Reducers.ts

```diff
@@ -16,16 +16,15 @@ import shortid from 'shortid';
 import { ReduxAction, ReduxConstants } from '../typings/ReduxConstants';
 import { initialState } from './State';
 
-
 // Tests if the expression already exists in the list
 const hasExpression = (
   list: ReadonlyArray<Expression>,
   action: { payload: Expression },
-) => list.some(expObj => expObj.expression === action.payload.expression);
+) => list.some((expObj) => expObj.expression === action.payload.expression);
 
 // Creates a new Expression object to be stored in the list
 const newExpressionObject = (
-  state: Expression | {},
+  state: Expression | Record<string, unknown>,
```

Comment:
Record is something new to me...I guess any object is technically a Record...as long as we know the type?

Reply:
Eslint recommended this change.